spec/requests/api/v1/attendances_spec.rb

```diff
@@ -4,24 +4,41 @@
 
 RSpec.describe 'Api::V1::Attendances', type: :request do
   let!(:logged_in_user) { create(:confirmed_user) }
-  let!(:business) { create(:business, user: logged_in_user) }
-  let!(:child) { create(:child, business: business) }
+  let!(:business)       { create(:business, user: logged_in_user) }
```

Comment:
Can you double check your rubocop or editor config? Consistently throughout the other specs we're using a single space instead of lining them up; would love to keep it consistent (but feel free to make a ticket to change that style!) - if rubocop doesn't automatically edit this, I'd love to see if it's something we can turn on, just so it's consistent across the app.

Reply:
Thanks for pointing that out. I actually don't have a preference, just a habit that leaked out 😄. Will look into that

Reply:
We can revisit code styles later